src/database/schema.go

```diff
@@ -22,33 +22,46 @@ func createTables() error {
 
 	_, err = DB.Exec(`
 		CREATE TABLE IF NOT EXISTS pending_donations (
-			token  UUID UNIQUE NOT NULL DEFAULT gen_random_uuid(),
+			token  UUID PRIMARY KEY DEFAULT gen_random_uuid(),
 			created_at BIGINT NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW())::BIGINT, -- unix seconds
 			paypal_order_id TEXT UNIQUE, -- can be null in case we want to make a "gift card" with no paypal order id attached
 			amount INTEGER, -- can be null for the same reason
 
 			used BOOL NOT NULL DEFAULT FALSE,
-			used_by UUID -- user_id, not mcuuid
+			used_by UUID REFERENCES users(user_id)
```

Comment:
yeah it's safe to do now, but i'm too lazy! we should manually add the constraint too, since the code won't actually execute unless the table is removed.